python/lsst/ts/observatory/control/maintel/mtcs.py

```diff
@@ -579,6 +579,37 @@ async def dome_el_in_position(self) -> str:
         await self._dome_el_in_position.wait()
         return "Dome elevation in position."
 
+    async def wait_for_dome_ready(self) -> None:
```

Comment:
This method is only checking that the dome is in enables state. 

We usually do that at a different layer, using the [assert_all_enabled](https://github.com/lsst-ts/ts_observatory_control/blob/9e9ab43b4474a69c4e12723776bfbb1c55e9a023/python/lsst/ts/observatory/control/remote_group.py#L759) method. 

I think what you want to do here is check that the dome is not parked already, and that it is in a state that can be parked. So probably check the azEnabled or azMotion events.

Reply:
I considered the azEnabled but looking at the mtdome_csc.py it looks to me they use it more for internal checks.  In the do_park command they check for the summary state to be Enabled. 

The other thing to consider is that in the mtcs park_dome method I check the dome is not parked already but In that  test I don't check for the inPosition state to be True. I do that later when we command the dome to park, then we check for the PARKED state and also to be in position.